README.md

```diff
@@ -144,6 +146,26 @@ $ sudo dokcer compose stop
 
 ## FAQ
 
+### Use Ascender without Docker
+
+We recommend using Ascender with Docker as described above. However, you might not be able to install Docker in your development environment due to permission issues or etc.
+
+In such cases, Ascender can be used without Docker. To do that, please install Poetry in your computer, and follow the steps describing in "Start development" section with ignoring the steps related to Docker.
+
+```bash
+# Install Poetry
+$ pip3 install poetry
+
+# Clone repo
+$ git clone git@github.com:cvpaperchallenge/<YOUR_REPO_NAME>.git
```

Comment:
```suggestion
$ git clone git@github.com:<YOUR_USER_NAME>/<YOUR_REPO_NAME>.git
```
IIUC, Ascender is used as template project. If so actual project is in under user repositories.